extensions/shared/src/main/java/app/revanced/extension/youtube/patches/utils/PlayerControlsPatch.java

```diff
@@ -68,6 +69,8 @@ private static void changeVisibility(boolean showing, boolean animation) {
 
         // CreateSegmentButtonController.changeVisibility(showing, animation);
         // VotingButtonController.changeVisibility(showing, animation);
+
+        SegmentPlaybackController.onPlayerControlsVisibilityChanged(showing);
```

Comment:
This Instruction must be added by the patch.

Otherwise, even if the `Overlay buttons` patch is included and the `SponsorBlock` patch is exclude, the function is invoked.